lib/test/CMakeLists.txt

```diff
@@ -260,7 +260,9 @@ ot_check_test (TimeSeries_std)
 ot_check_test (FilteringWindows_std)
 ot_check_test (WelchFactory_std)
 ot_check_test (ExponentialModel_std)
+if (Boost_FOUND)
 ot_check_test (CovarianceModel_std IGNOREOUT)
+endif ()
```

Comment:
Why `boost` is required here?

Reply:
MaternModel uses some bessel functions and an assert fails

Reply:
Thus maybe we need to update the assert? there is an alternative to `Bessel` functions inside the library and it would be pity if test is not performed because `boost` is missing. What do you think?

Reply:
it's something else: this test randomly fails unless I disable tbb

Reply:
yes: LogBesselK uses SymbolicFunction without boost so it's not thread-safe

Reply:
I can provide an implementation without SymbolicFunction if needed. Would it help?

Reply:
I already implemented MaternModel::isParallel